datapage.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: data_jp
+title: "Data"
+description: "C-PROOF Glider Data"
+header-img: "img/DataPageHeaderPhoto.jpg"
+---
+<html>
+<style>
+    table{
+        background-color: rgb(250, 250, 250) ;
+        width: 100%;
+        text-align: left;
+
+    }
+</style>
+<body class="datapage"> 
+
+
+<h2>Access Data</h2>
+
+<table >
+  <tr>
+    <td>
+      <p>
+        <a href="{{ site.baseurl }}/data.html"><strong>Direct download</strong></a>: Wget the data for all missions, by glider line, or by platform </br>
+        <a href='{{ site.baseurl }}/deployments/index.html'><strong>Deployments</strong></a>: View and download data and figures for individual glider missions
+      </p>
+    </td>
+  </tr>
+ 
+</table>
+
+
+<h2>Data Description</h2>
+
+The different types of data files that are available are named as follows:
+<ul>
+    <li>Realtime: Subset of data transmitted via Iridium while the glider is in mission, with basic automated processing steps and QA/QC applied. </li>
+    <li>Delayed: Full dataset collected by the glider and recovered post-mission, with all processing steps and QA/QC applied. </li>
+    <li>Level 0 (L0) data: Merged raw data files, with no interpolation and no processing. </li>
+    <li>Level 1 (L1) data: Time series or profile data with some interpolation and calculation of additional variables like salinity. </li>
+    <li>Level 2 (L2) data: Gridded level 1 data, on a depth-time grid. </li>
+</ul>
+
+
+
+
+    <h2>Data Processing  </h2>
+
+    <table style="width:100%">
+      <tr>
+        <td><a href="test.pdf" target="_blank">Open PDF</a></td>
```

Comment:
Let's remove the blank PDFs for now (just comment out if possible), so they don't look like broken links.